core/tests/protractor_desktop/additionalEditorFeaturesModals.js

```diff
@@ -29,6 +29,7 @@ var ExplorationEditorPage =
   require('../protractor_utils/ExplorationEditorPage.js');
 
 var lostChangesModal = element(by.css('.protractor-test-lost-changes-modal'));
+var saveChangesButton = element(by.css('.protractor-test-save-changes'));
```

Comment:
We shouldn't be selecting elements in test files. All page interactions should go through the utilities in `protractor_utils/`

Reply:
Done.